### DISCUSSION TITLE
Files dock icon looks very similar to "Show Desktop" icon

### DISCUSSION THREAD

**Discussion by sryze** [elementary/files#2209] [Ideas]
Screenshot:

![Screenshot from 2023-05-27 15 19 23](https://github.com/elementary/files/assets/4525736/a116c274-c9a6-44f1-9e20-e3b94c0f8212)

It's easy to sometimes click on the wrong icon when you are in a hurry. In previous versions the Files icon had a different color so this wasn't an issue.

**Comment by danirabbit** [comment]
I'm going to close this as "out of scope" since this seems more like an issue with an unsupported plank applet than with Files
